scanner/windows_test.go

```diff
@@ -415,6 +443,62 @@ func Test_detectOSName(t *testing.T) {
 			},
 			want: "Windows Server 2019",
 		},
+		{
+			name: "Windows Server 2022",
+			args: osInfo{
+				productName:      "Windows Server",
+				version:          "10.0",
+				build:            "20348",
+				revision:         "",
+				edition:          "",
+				servicePack:      "",
+				arch:             "x64-based",
+				installationType: "Server",
+			},
+			want: "Windows Server 2022",
+		},
+		{
+			name: "Windows Server 2022, 23H2 Edition",
+			args: osInfo{
+				productName:      "Microsoft Windows Server 2022 Datacenter",
+				version:          "10.0",
+				build:            "25398",
+				revision:         "",
+				edition:          "",
+				servicePack:      "",
+				arch:             "x64-based",
+				installationType: "Server",
+			},
+			want: "Windows Server 2022, 23H2 Edition",
+		},
+		{
+			name: "Windows Server 2025",
+			args: osInfo{
+				productName:      "Microsoft Windows Server 2022 Datacenter",
+				version:          "10.0",
+				build:            "26100",
+				revision:         "",
+				edition:          "",
+				servicePack:      "",
+				arch:             "x64-based",
+				installationType: "Server",
+			},
+			want: "Windows Server 2025",
+		},
+		{
+			name: "Windows Server latest release",
+			args: osInfo{
+				productName:      "Microsoft Windows Server 2022 Datacenter",
```

Comment:
This is not related to OS detection logic, but s/2022/2025/ is better ?

Reply:
fixed: https://github.com/future-architect/vuls/compare/3ff0a9386427683eeb17b825d5dc1f8017242587..daa53643dc46493ef07ca2faa97f559576d0ddf3